.github/workflows/adbe-unittests-python2.yml

```diff
@@ -0,0 +1,28 @@
+name: AdbeUnitTestsPython2.7
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        api-level: [21, 26, 29]
+        target: [default] # Other option: google_apis
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: run tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: ${{ matrix.api-level }}
+          target: ${{ matrix.target }}
+          arch: x86_64
```

Comment:
FYI Google recommends only using x86_64 images for API 26+. In my experience the the x86_64 images are slightly less stable especially on lower API levels and the performance difference vs x86 is hard to notice on CI.

Reply:
Thanks. I will keep x86_64 for now, the tests are fast enough that as long as it is fast, it is fine.